tests/unit/src/main/scala/tests/Library.scala

```diff
@@ -25,95 +26,49 @@ object Library {
   def cats: Seq[AbsolutePath] =
     fetch("org.typelevel", "cats-core_2.12", "2.0.0-M4")
   def all: List[Library] = {
-    val settings = new Settings()
+    val fetch = Fetch
+      .create()
+      .withMainArtifacts()
+      .withClassifiers(Set("sources", "_").asJava)
       .withDependencies(
-        List(
-          new Dependency(
-            "org.scalameta",
-            "scalameta_2.12",
-            "4.1.4"
-          ),
-          new Dependency(
-            "com.lihaoyi",
-            "scalaparse_2.12",
-            "2.1.0"
-          ),
-          new Dependency(
-            "com.lihaoyi",
-            "acyclic_2.12",
-            "0.1.8"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-testkit_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-stream_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-cluster_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "org.apache.spark",
-            "spark-sql_2.11",
-            "2.2.1"
-          ),
-          new Dependency(
-            "org.eclipse.jetty",
-            "jetty-servlet",
-            "9.3.11.v20160721"
-          ),
-          new Dependency(
-            "org.apache.kafka",
-            "kafka_2.12",
-            "1.0.0"
-          ),
-          new Dependency(
-            "org.apache.flink",
-            "flink-parent",
-            "1.4.1"
-          ),
-          new Dependency(
-            "io.grpc",
-            "grpc-all",
-            "1.10.0"
-          ),
-          new Dependency(
-            "org.scala-lang",
-            "scala-compiler",
-            mtags.BuildInfo.scalaCompilerVersion
-          ),
-          new Dependency(
-            "io.buoyant",
-            "linkerd-core_2.12",
-            "1.4.3"
-          )
-        )
+        Embedded.dependency(
+          "org.scalameta:scalameta_2.12:4.1.4",
```

Comment:
If the order doesn't matter then these should be listed alphabetically.

Reply:
Sorted.